source/includes/azure/_disks.md

```diff
@@ -245,3 +245,53 @@ curl -X POST \
 <code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/disks/:id?operation=detach</code>
 
 Detach an existing disk from a given [instance](#azure-instances).
+
+<!-------------------- ATTACH A DISK -------------------->
+
+#### Attach a disk
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure-conn/test_env/disks/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/the-sheep_disk1_c0cbe27c49aa928525e93bd2519aac4?operation=attach"
```

Comment:
Can we change the url path to be a bit more generic instead of using our real IDs and names?

Reply:
ab36c1c83d0980c67fd84a0f0cafeaace182f146
changed it here